### DISCUSSION TITLE
Erratic window positioning behavior after running PW

### DISCUSSION THREAD

**Discussion by hobbesVT** [kangyu-california/PersistentWindows#199] [Q&A]
I've tried to see if PW solves my issues with windows jumping to the main monitor (I have 3) and gave it a try.
Specifically one application (emClient, which is my mail program) opens child windows (e.g. when composing a new message) in exactly the same place where the main window resides. Before running PM, Windows 11 remembered the position of child windows of apps.
Seems like I've traded one unwanted behavior for another ...

Now, since I stopped PM (exited, not running in task manager) app windows still open where they were under PM, and Windows 11 does not remember any repositioned windows any more. 
What has changed in the registry that could have stopped that ability?
I've deleted the db directory that PM created on my hard drive - but to no avail - app window positions are not stored any more by the OS.

Can anyone point me to the registry where such an option could have been deactivated?
Any other idea what could be going on?
Thanks,
Georg

**Comment by kangyu-california** [comment]
PW does not use or depend on registry.
The only place where PW stores persistence information is C:\Users\[User]\AppData\Local\PersistentWindows

The issue seems to be on Windows 11 side, which is beyond my knowledge.

**Reply by hobbesVT** [reply]
Thanks for the quick reply confirming PW only writes into appdata - must be something else going on.
May have to do with fancy zones even though I didn't have issues there before, other than that my open windows huddled on the main screen after sleep at times - 3 monitors on a dell D3100 with two HDMI and one DP seem to come out of sleep at different times.
I'll update here if I find out anything else.

**Comment by hobbesVT** [comment]
I tested some different scenarios now with different monitor and display configurations. It seems the issues are related to fancy zones, which are part of Window 11's PowerToys. It looks to me (not an expert windows programming) as if fancy zones assigns numbers to zones that act like screens or entire displays in some way. The issue was specifically there when a window was maximized to a zone, and then all child windows also opened maximized to the same zone. I usually prefer child windows on the same display with partial overlap so that I can switch easily focus between them. E.g. my  email client occupies a zone on my display, but the new mail child window is offset to that zone but on the same display. This way I can click on either header of the window (or other area that protrudes out) to switch focus. This is the sequence on how I got this back:
- snap main window to zone where it should open by default (with fancy zones on)
- then turn fancy zones off 
- now open any child window (e.g. new mail. edit views of app) - they will first open to the parent zone from previous memory.
- reposition and resize the child windows on the display as you like
- then close child windows first, then the main application
- now you can turn fancy zones on again -> the main application will open full size in the zone, and child windows will open where you set them.

Sorry that I thought PW was the culprit of the issues first. Hope that helps someone else!

**Reply by kangyu-california** [reply]
I would like to run both PW and FZ.
Let FZ lead the way by creating the initial zone setting, and PW will automatically follow the setting.
This way user get the best of both tools.
